Cargo.toml

```diff
@@ -30,6 +30,7 @@ nispor = "1.2.7"
 clap = { version = "3.0.12", features = ["derive"] }
 env_logger = "0.9.0"
 http = "0.2.8"
+macaddr = "1.0.1"
```

Comment:
in netavark we just use Vec<u8> instead of separate create and convert them with https://github.com/containers/netavark/blob/b12c33f49353e2cef6617e7f32b0401234b87ee2/src/network/core_utils.rs#L167-L202